test/test_stdout.rb

```diff
@@ -18,4 +18,9 @@ def test_default_list_output
       assert_match(/.*#{spec_key}.*/, @stdout.string)
     end
   end
+
+  def test_ubuntu_host_generation_output
```

Comment:
This doesn't really test the changes in this PR, it's just testing that this message is printed to stderr.

I think we want to test that stdout/stderr can be redirected in Ruby, which is what your PR is making possible.

So I'd think our test would want to override `$stdout`/`$stderr`, call `CLI.execute!` (with a much-simpler input; we don't care that it's ubuntu or anything about the input actually), then assert that the variables contain the message.